docs/examples/http_to_influx.json

```diff
@@ -11,7 +11,7 @@
   "handlers": {
     "jsontemplating": {
       "parser": "json",
-      "transformers": [ "templater" ],
+      "transformers": [],
```

Comment:
Considering the name, if we remove the templater transformer, we should rename the handler as well.

Could also just specify a templater transformer.